staging/src/k8s.io/apiserver/pkg/apis/audit/v1beta1/types.go

```diff
@@ -156,6 +156,11 @@ type Policy struct {
 	// The default audit level is None, but can be overridden by a catch-all rule at the end of the list.
 	// PolicyRules are strictly ordered.
 	Rules []PolicyRule `json:"rules" protobuf:"bytes,2,rep,name=rules"`
+
+	// OmitStages specify events generated in which stages will not be emitted to backend.
```

Comment:
Having difficulties to parse this sentence (same for the one in the rules). Isn't this simpler:

OmitStages are a list of stages for which no events are created. Note that this can also
be specified per rule in which case the union of stages is skipped.

Reply:
Done.